app/views/datasets/show.html.erb

```diff
@@ -75,7 +75,7 @@
             <div role="note" aria-label="Publisher information" class="panel panel-border-narrow text">
               <p><%= t('datasets.publishers.information') %></p>
             </div>
-            <p><%= link_to t('datasets.publishers.button'), "/dataset/edit/#{@dataset.legacy_name}", role: 'button', class: 'button', rel: %w(nofollow) %></p>
+            <p><%= link_to t('datasets.publishers.button'), "#{Plek.find("ckan")}/user/login", role: 'button', class: 'button', rel: %w(nofollow) %></p>
```

Comment:
At least my interpretation of this was that the change was intended for only users who aren't logged in to CKAN, whereas this will affect all users, sending them to the login page, rather than the page for the dataset they want to edit?

Reply:
That sounds possible. If I had a CKAN account I could test what currently happens when users are logged in and click that button... 👀

Reply:
The button text is still "Sign in" whether or not you're signed in. 🤔I'll go back to the drawing board on this!

Reply:
At least from the planning meeting, this sounded more like "handle unauthenticated requests for editing datasets in CKAN by redirecting to the login page".

Reply:
Yep, totally. Having re-read the card, it should be done in CKAN. I think the "Sign in" button not linking to the right page was a red herring.